index-africa

```diff
@@ -4,27 +4,47 @@ website: https://africa.usegalaxy.eu
 subdomain: Africa
 ---
 
-# Welcome to africa.usegalaxy.eu, the African galaxy server
+![African Galaxy](<img src="https://avatars0.githubusercontent.com/u/68950141?s=200&v=4" height="100px" alt="ASaiM logo"/>){:.sc-intro-left}
```

Comment:
"ASaiM logo" should be fixed.

Reply:
Yeah! I needed to move home... So I decided to keep the commits... I'm on it... I have not completed the edits